packages/testing/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@usedapp/testing",
+  "version": "0.1.0",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "repository": "git@github.com:EthWorks/dapp-framework.git",
+  "author": "Ethworks",
+  "license": "MIT",
+  "devDependencies": {
+    "@ethereum-waffle/provider": "^3.2.2",
+    "@ethersproject/abi": "^5.0.12",
+    "@ethersproject/bignumber": "^5.0.14",
+    "@ethersproject/contracts": "^5.0.10",
+    "@testing-library/react-hooks": "^5.0.3",
+    "@types/chai": "^4.2.14",
+    "@types/mocha": "^8.2.0",
+    "@typescript-eslint/eslint-plugin": "^4.10.0",
+    "@typescript-eslint/parser": "^4.10.0",
+    "@web3-react/abstract-connector": "^6.0.7",
+    "@web3-react/types": "^6.0.7",
+    "chai": "^4.2.0",
+    "eslint": "7.19.0",
+    "jsdom": "^16.4.0",
+    "jsdom-global": "^3.0.2",
+    "mocha": "^8.2.1",
+    "prettier": "^2.1.2",
+    "typescript": "^4.1.3"
+  },
+  "dependencies": {
+    "@ethersproject/providers": "^5.0.17",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@usedapp/core": "*",
+    "@web3-react/core": "^6.1.1",
+    "@web3-react/injected-connector": "^6.0.7",
```

Comment:
Is that dependency needed?

Reply:
https://github.com/EthWorks/useDapp/pull/42